src/index/ivf/ivf.cc

```diff
@@ -320,15 +319,13 @@ IvfIndexNode<T>::Train(const DataSet& dataset, const Config& cfg) {
             index = std::make_unique<faiss::IndexIVFScalarQuantizer>(qzr, dim, nlist, faiss::QuantizerType::QT_8bit,
                                                                      metric.value());
             index->train(rows, (const float*)data);
-            index->make_direct_map(true);
```

Comment:
We don't need this for SQ and PQ

Reply:
limited the make operation by having raw data or not